apps/gauzy/src/app/pages/integrations/components/plugin-builtin/plugin-builtin.component.html

```
@@ -0,0 +1,102 @@
+		}
+		<nb-select
+			fullWidth
+			[placeholder]="'INTEGRATIONS.FILTER_INTEGRATIONS' | translate"
+			[selected]="selectedIntegrationFilter$ | async"
+			(selectedChange)="setSelectedIntegrationFilter($event)"
+		>
+			@for (filter of filters; track filter.value) {
+				<nb-option [value]="filter.value">{{ filter.label }}</nb-option>
+			}
+		</nb-select>
+		<button (click)="clearFilter()" status="danger" size="small" nbButton>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)